database/migrations/20220407074127-create-whitelist.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("whitelists", {
```

Comment:
nit: could we change the table name to be singular as this is a singular whitelist of emails

Reply:
Makes sense, changed in https://github.com/isomerpages/isomercms-backend/pull/422/commits/61220d8803326153f6854fd9c85a13caac9ed31d